src/group/views.py

```diff
@@ -85,6 +87,76 @@ def post(self, request, pk):
         return success_response(serializer.data)
 
 
+class GroupShowsAdd(generics.GenericAPIView):
```

Comment:
could you put the shows add and shows remove together ? they're currently separated by groupmembers add and groupmembersremove

Reply:
changed the ordering!